package.json

```diff
@@ -43,8 +43,7 @@
     "babel-plugin-module-resolver": "^4.0.0",
     "bower-config": "^1.4.3",
     "bower-endpoint-parser": "0.2.2",
-    "broccoli": "^3.4.2",
-    "broccoli-amd-funnel": "^2.0.1",
+    "broccoli": "git+https://github.com/broccolijs/broccoli.git#0c56d222634097542f571ff8a79532f8580e3c0c",
```

Comment:
Ready for update to https://github.com/broccolijs/broccoli/releases/tag/v3.5.0

Reply:
Updated to the released version, and fixed a mistake where I dropped a line from package.json unintentionally.